subworkflows/local/bgc.nf

```diff
@@ -0,0 +1,65 @@
+/*
+    Run AMP screening tools
+*/
+
+include { UNTAR as UNTAR_CSS                       } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_DETECTION                 } from '../../modules/nf-core/modules/untar/main'
+include { UNTAR as UNTAR_MODULES                   } from '../../modules/nf-core/modules/untar/main'
+include { ANTISMASH_ANTISMASHLITEDOWNLOADDATABASES } from '../../modules/nf-core/modules/antismash/antismashlitedownloaddatabases/main'
+include { ANTISMASH_ANTISMASHLITE                  } from '../../modules/nf-core/modules/antismash/antismashlite/main'
+
+workflow BGC {
+
+    take:
+    fna     // tuple val(meta), path(PROKKA.out.fna)
+    gff     // tuple val(meta), path(PROKKA.out.gff)
+
+    main:
+    ch_versions = Channel.empty()
+
+    if ( !params.bgc_skip_antismash ) {
+        // Check whether user supplies database and/or antismash directory. If not, obtain them via the module antismashlite/antismashlitedownloaddatabases.
+        // Important for future maintenance: For CI tests, only the "else" option below is used. Both options should be tested locally whenever the antiSMASH module gets updated.
+        if ( params.bgc_antismash_databases && params.bgc_antismash_installationdirectory ) {
+
+            ch_antismash_databases = Channel
+                .fromPath( params.bgc_antismash_databases )
+                .first()
+
+            ch_antismash_directory = Channel
+                .fromPath( params.bgc_antismash_installationdirectory )
+                .first()
+
+        } else {
+
+            log.warn "Warning: No antiSMASH database and/or directory supplied – they will be downloaded by the pipeline."
```

Comment:
Actually, this should be moved to alongside here: https://github.com/nf-core/funcscan/blob/0667b793801d96547594048ee14da0892946384a/workflows/funcscan.nf#L34

You odn't want to get halfway through the pipeline to realse you forogt to supply the database... you will need to copy the duplicate as well

Reply:
Thanks! Makes sense, I just didn't notice time-wise difference (because the test is very fast anyway).